css/properties/z-index.json

```
@@ -40,6 +40,38 @@
             "deprecated": false
           }
         },
+        "auto": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
```

I know we're not maintaining IE but this data is very likely wrong. Can you omit IE data?

```suggestion
                "version_added": "4"
```

---

Unfortunately our linter prevents us from omitting IE data since it is a desktop browser, so we'll have to tweak the linter.  Should we do so?

---

Not sure, I think very often this would be the same IE version as for the parent feature or `false`.